docs/tutorials/Raster-data-Lambert93.md

```diff
@@ -20,20 +20,23 @@ The webpage we want to display data on should be structured as follows :
     </head>
     <body>
         <div id="viewerDiv"></div>
-        <script src="js/itowns.js"></script>
+        <script src="../dist/itowns.js"></script>
```

Comment:
Didn't see lines 38 and 39 when writing this. We could therefore also leave it as such. WDYT between this and a generic path ?

Reply:
I would leave it like this since it is easier for people cloning the repository and testing in the examples folder (no webserver to configure, easy to try on) and people that want to adapt it know how to do it with lines 38-39. Is that ok for you?

Reply:
Yep, fine for me :+1: